From b6376e83945fa7343ee77250a255bc4eb1ae0d6b Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Date: Fri, 6 Mar 2020 12:55:01 +0200
Subject: [PATCH 110/269] s32g274a: Remove unnecessary condition

Remove superfluous "#if IMAGE_BL31" from bl31 file.

Upstream-Status: Pending 

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
Issue: ALB-4910
---
 plat/s32g/s32g275_bl31.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/plat/s32g/s32g275_bl31.c b/plat/s32g/s32g275_bl31.c
index 5ebfa0f6f..fbe6426e4 100644
--- a/plat/s32g/s32g275_bl31.c
+++ b/plat/s32g/s32g275_bl31.c
@@ -166,9 +166,7 @@ static void s32g_el3_mmu_fixup(void)
 
 void s32g_gic_setup(void)
 {
-#if IMAGE_BL31
 	gicv3_driver_init(&s32g275_gic_data);
-#endif
 	gicv3_distif_init();
 	gicv3_rdistif_init(plat_my_core_pos());
 	gicv3_cpuif_enable(plat_my_core_pos());
-- 
2.17.1

